.travis.yml

```diff
@@ -3,7 +3,11 @@ rvm:
 - 2.2.2
 gemfile:
 - spec/dummy/Gemfile
-install: bundle install
+install:
+- bundle install
+- cd spec/dummy && bundle install
```

Comment:
@mapreal19 We're doing a lot of reworking of the internals, so I'd first focus on the capybara tests. Especially important to get a test that verifies the created HTML for server rendering.

You might want to use the same driver we use in https://github.com/shakacode/react-webpack-rails-tutorial